spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    # sequence(:email) { |n| "user#{n}@gmail.com"}からの書き換え
+    sequence(:email, "user1@sample.com")
+    password {"password"}
+    password_confirmation {"password"}
```

Comment:
波括弧（ブレース）の中はスペースで区切った方が見やすいですね。
```suggestion
    password { "password" }
    password_confirmation { "password" }
```

Reply:
確かにそちらの方が見やすいですね。修正します。

Reply:
修正の際はこちらのコメント欄に対応するコミットIDを半角スペース区切りで返答する形でお願いします。

Reply:
こちらで修正しました。
[84c91b7](https://github.com/KozakuraRyuuichi/sample_app_for_rspec/pull/2/commits/84c91b7e930dbc28facf1851e00d2be8801c99d4)